### PR TITLE
Update PHP 7 support

### PR DESCRIPTION
**Pull request by SenseException**

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          - "7.3"
+          - "7.4"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP with Xdebug"
```
Comment (greg0ire) [inline .github/workflows/continuous-integration.yml]:
```suggestion
      - name: "Install PHP with PCOV"
```

**Comment by SenseException** [comment]
@greg0ire Do you think I can change this PR to use `1.0` as the base branch or do you think that support of newer PHP versions should happen in the next minor version?

**Comment by greg0ire** [comment]
Since it's not a breaking change, I would say 1.0 should be the base branch 👍

**Comment by SenseException** [comment]
The GitHub Actions build is gone after I changed the base branch in the PR inside my fork. I can't show that the workflows are working. Recreating the PR didn't work either. :-/

**Comment by greg0ire** [comment]
Let me setup the automatic release workflow, it will help.

**Comment by greg0ire** [comment]
That and a rebase.

**Comment by greg0ire** [comment]
Thanks @SenseException !

**Comment by SenseException** [comment]
Thank you too

**Pull request merged**